x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/pagerduty.tsx

```diff
@@ -237,6 +269,33 @@ const PagerDutyParamsFields: React.FunctionComponent<ActionParamsProps<PagerDuty
                 defaultMessage: 'DedupKey (optional)',
               }
             )}
+            labelAppend={
```

Comment:
There is a lot of duplication for these popovers.  It _seems_ like they could be refactored to function call that returned the popover.  Maybe not.  But if they could be refactored, recommend opening a tech debt issue to resolve later.

Reply:
Yeah. Thank you for the catch! I fixed this by move to a single function.